e2e/utils.go

```diff
@@ -1207,3 +1211,43 @@ func addTopologyDomainsToDSYaml(template, labels string) string {
 	return strings.ReplaceAll(template, "# - \"--domainlabels=failure-domain/region,failure-domain/zone\"",
 		"- \"--domainlabels="+labels+"\"")
 }
+
+// createCustomConfigMap provides multiple clusters information.
+func createCustomConfigMap(pluginPath string, c kubernetes.Interface, clusterID1, clusterID2 string) {
```

Comment:
instead of sting pass a map with key cluster and subvolumegroupname as the value.

Reply:
@Madhu-1 Since we will require keys of the map too, will passing only a map and retrieving the keys afterwards increase the complexity? Also, you are right, Passing the map along with strings will remove the hardcoded nature of setting subvolume group name in utils. What should be a better approach?

Reply:
better to pass the map as of now.